skyportal/handlers/source.py

```diff
@@ -114,7 +114,12 @@ def post(self):
         try:
             group_ids = data.pop('group_ids')
```

Comment:
Is there any verification done to ensure that a user doesn't add sources to a group they don't belong to?